pkg/types/validation/installconfig.go

```diff
@@ -63,6 +64,18 @@ func ValidateInstallConfig(c *types.InstallConfig, openStackValidValuesFetcher o
 	nameErr := validate.ClusterName(c.ObjectMeta.Name)
 	if nameErr != nil {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("metadata", "name"), c.ObjectMeta.Name, nameErr.Error()))
+	} else if c.Platform.OpenStack != nil {
+		fmt.Println("Openstack Validation")
+		// number of digits in the number of replicas specified
+		replicaDigits := math.Floor(math.Log10(float64(*c.ControlPlane.Replicas))) + 1
```

Comment:
If I understood the logic correctly it can be rewritten with Ceil like
replicaDigits := math.Ceil(math.Log10(float64(*c.ControlPlane.Replicas)))